python/pyngraph/serializer.cpp

```diff
@@ -30,8 +30,7 @@ void regclass_pyngraph_Serializer(py::module m)
                           bool binary_constant_data = false);
```

Comment:
Can you update the comment as well? "bool binary_constant_data = false" is not an argument anymore.

Reply:
binary_constant_data still exists on a private version of this function. The user can't ever set binary_constant_data to true so I removed the parameter. This parameter is used with one of the other serializer functions that stores the serialized data in a cpio file (kind of like a tar file). It was never intended to be in the public API and so it was removed.